pyproject.toml

```diff
@@ -64,6 +65,7 @@ optional = true
 black = "^24.3.0"
 radon = "^6.0.1"
 mypy = "^1.9.0"
+types-jsonschema = "^4.23.0"
```

Comment:
Out of curiosity, where is this used?

I'm asking because upstream we've been trying to improve JSON Schema support:

- https://github.com/IQSS/dataverse-pm/issues/26

Reply:
In utils, jsonschema.validate is used:

https://github.com/gdcc/pyDataverse/blob/3b6fe063dfc73d6fa3aa674cc02313504c40204f/pyDataverse/utils.py#L8

jsonschema is not typed, so mypy requires the type stubs for it. When you run mypy on the code you will get the respective error.

Reply:
I had a look, the call chain is: `pyDataverse.models.(Dataset|DVObject).validate_json` -> `pyDataverse.utils.validate_data` -> `jsonschema.validate`.
There are also four schema.json files checked in at pyDataverse/schemas/json (which is not a package though, so one would need the filepaths to validate against them).

The function is not called automatically and supposed to be usable by consumers of the package, the validate_json functions are tested in test_datafile.py, test_dataset.py, and test_dataverse.py. The paths to the schemas are declared in the conftest.py.